test/e2e/storage/persistent_volumes-disruptive.go

```diff
@@ -108,6 +109,107 @@ var _ = SIGDescribe("PersistentVolumes[Disruptive][Flaky]", func() {
 		framework.DeletePodWithWait(f, c, nfsServerPod)
 	})
 
+	Context("when kube-controller-manager restarts", func() {
+		var (
+			diskName1, diskName2 string
+			err                  error
+			pvConfig1, pvConfig2 framework.PersistentVolumeConfig
+			pv1, pv2             *v1.PersistentVolume
+			pvc1, pvc2           *v1.PersistentVolumeClaim
+			clientPod            *v1.Pod
+		)
+
+		BeforeEach(func() {
+			framework.SkipUnlessProviderIs("gce", "gke")
```

Comment:
I think we need to skip the test for gke because we will not be able to log into the master.